srcDoc/release_notes.rst

```diff
@@ -62,6 +62,22 @@ following command-line parameters:
     --cpp-build
     --cs-build
 
+What's Fixed in Master
+~~~~~~~~~~~~~~~~~~~~~~
+
+Review Dynamic Data Send() and Receive() operations (#55)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*RTI Perftest* has optimize the `Send()` and `Received()` methods by:
```

Comment:
has optimized*

but I would say:

The `Send()` and `Receive()` functions for *Dynamic data* have been optimized by: